src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/PrisonApi.java

```diff
@@ -358,20 +358,6 @@ public List<ScheduledAppointmentSearchDto> getScheduledAppointments(final String
                 .block();
     }
 
-
-    public List<ScheduledAppointmentDto> getScheduledAppointments(final String agencyId, final LocalDate date) {
```

Comment:
Assume if this is being removed that nothing else is using it?

Reply:
Yes it is not used